### PR TITLE
Add documentation for {when_iso}.

### PR DESCRIPTION
**Pull request by maruel**
Clarify that {when} is in local time.

### CODE REVIEW

**Comment by maruel** [comment]
Related to https://github.com/mholt/caddy/pull/1353

File: content/docs/placeholders.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ These values are obtained from the request.
 *   **{scheme}** - The protocol/scheme used (usually http or https)
 *   **{uri}** - The request URI (includes path, query string, and fragment)
 *   **{uri_escaped}** - The query-escaped variant of {uri}
-*   **{when}** - Timestamp in the format `02/Jan/2006:15:04:05 -0700`
+*   **{when}** - Timestamp in the format `02/Jan/2006:15:04:05 -0700`in local time
+*   **{when_iso}** - Timestamp in the format `2006-01-02T15:04:05Z in UTC
```
Comment (mholt) [inline content/docs/placeholders.md]: This code block should be closed (after the timestamp) but I can merge it and fix it myself later, either way.

Comment (maruel) [inline content/docs/placeholders.md]: Oops fixed.

**Comment by mholt** [comment]
Great, thank you! 🙌

**Pull request merged**